### ISSUE TITLE
Is it safe to use hacked and not intended features.

### ISSUE THREAD

**Issue by ggenije** [LLK/scratch-vm#3752]
I wonder is it safe in terms, do you planning it removing support for it in any point in future for hacked blocks,
especially for "while" block.
Also I found out that you can play "item of (list)" as boolean input, which is pretty useful for me, but I'm afraid you could remove this feature, which would ruin my projects.
Also it would be also useful you can place variables also in boolean inputs.

**Comment by PPPDUD** [comment]
I am not an ST member, but I have used a certain glitch to see if a sprite is touching a clone of itself, and it hasn't broken yet.

**Comment by griffpatch** [comment]
I think there are some relatively safe hacked blocks, like the touching
self trick.
But from experience, when we moved from Scratch 2 to Scratch 3, a lot of
hacked blocks stopped working and some of my projects won't load to fix
them... So, I'd advise not including them in any project that you would
mind losing some time down the road.

**Comment by benjiwheeler** [comment]
We don't officially support any "hacked blocks" functionality. That said, we generally want to keep projects working, and when we transitioned to Scratch 3, we did spend a bunch of time making sure that some of the common forms of hacked blocks and unintended features kept working. (Sorry @griffpatch that we missed a bunch, or decided not to support some.) It's also probably going to be a long time before there's an in-depth rebuild of the VM along the lines of "Scratch 4". So while it's not something you want to count on forever, my informal advice is not to worry about it :)
